queries/dune_v2/period_slippage.sql

```diff
@@ -1,7 +1,8 @@
 -- https://github.com/cowprotocol/solver-rewards/pull/169
 with
-filtered_trades as (
+batch_meta as (
```

Comment:
This is only needed for debugging, right? But not strictly needed? Do you know how much query-time is spent on this?

Reply:
It is used later for many reasons but was being copied redundantly into every transfer row
For example:
- we use `dex_swaps`  in the buffer trade approximations
- we use `solver_address` for the slippage aggregation...
- There are lots of reasons why we need this but ZERO reasons  the data needs to be duplicated in every row for each transaction.